python/src/pymain.cc

```diff
@@ -1,19 +1,9 @@
 #include <boost/python.hpp>
-#include "options.cc"
-#include "Algebra.h"
-#include "Catalog.h"
-#include "Greyscale.h"
-#include "HeuristicOptimizer.h"
-#include "Visitor.h"
-#include "Coordinator.h"
-#include "extension.h"
-#include "reference.h"
-#include "options.h"
-#include "LightField.h"
+#include "PythonLightField.h"
 #include "number.h"
 
 
-namespace Python {
+namespace lightdb::python {
```

Comment:
Why is SetUpLocalEnvironment needed?

Reply:
What is the definition of `ExecuteLightField` (and ExecuteLightFieldAndOptimizer) needed for?  Wouldn't `static_cast`s be sufficient?